plottr/apps/inspectr.py

```diff
@@ -669,5 +666,8 @@ def script() -> None:
     parser = argparse.ArgumentParser(description='inspectr -- sifting through qcodes data.')
     parser.add_argument('--dbpath', help='path to qcodes .db file',
                         default=None)
+    parser.add_argument("--console-log-level",
+                        choices=("ERROR", "WARNING", "INFO", "DEBUG"),
+                        default="WARNING")
     args = parser.parse_args()
-    main(args.dbpath)
+    main(args.dbpath, args.console_log_level)
```

Comment:
shouldn't the value of the argument be converted to int that corresponds to the level?

Reply:
but the types should be updated then to match this

Reply:
oh, cool, didn't know!

it's fine either way :)